controls/radheatmap/tooltip.md

```diff
@@ -0,0 +1,148 @@
+
+#### __[C#] Example 3: Populating the RadHeatMap control__
+{{region radheatmap-tooltip-3}}
+	public partial class MainWindow : Window
+    {
+        public MainWindow()
+        {
+            this.InitializeComponent(); 
+			this.PrepareData();
+        }
+		
+		private void PrepareData()
+		{
+			int year = 2018;
+			string[] months = new string[6] { "Jan", "Feb", "Mar", "Apr", "May", "Jun" };
+			var randomNumberGenerator = new Random();
+
+			var source = new ObservableCollection<TempInfo>();
+			for (int i = 0; i < months.Length; i++)
+			{
+				for (int k = 0; k < 6; k++)
+				{
+					var info = new TempInfo() { Year = year + k, Month = months[i], Temperature = randomNumberGenerator.Next(10, 300) };
+					source.Add(info);
+				}
+			}
+
+			this.categoricalDefinition.ItemsSource = source;
+		}
+    } 
+{{endregion}}
+
+#### __[VB.NET] Example 3: Populating the RadHeatMap control__
+{{region radheatmap-tooltip-4}}
+    Public Partial Class MainWindow
+		Inherits Window
+
+		Public Sub New()
+			Me.InitializeComponent()
+			Me.PrepareData()
+		End Sub
+
+		Private Sub PrepareData()
+			Dim year As Integer = 2018
+			Dim months As String() = New String(5) {"Jan", "Feb", "Mar", "Apr", "May", "Jun"}
+			Dim randomNumberGenerator = New Random()
+			Dim source = New ObservableCollection(Of TempInfo)()
+
+			For i As Integer = 0 To months.Length - 1
+
+				For k As Integer = 0 To 6 - 1
+					Dim info = New TempInfo() With {
+						.Year = year + k,
+						.Month = months(i),
+						.Temperature = randomNumberGenerator.[Next](10, 300)
+					}
+					source.Add(info)
+				Next
+			Next
+
+			Me.categoricalDefinition.ItemsSource = source
+		End Sub
+	End Class
+{{endregion}}
+
+After having the sample data ready, we can go in the XAML code and define the RadHeatMap and the CellLabelSettings.
+
+#### __[XAML] Example 4: Customizing the ToolTip__
+{{region radheatmap-labels-5}}
+	<telerik:RadHeatMap ShowToolTips="True" CellBorderColor="Black" CellBorderThickness="1" >
+		<telerik:RadHeatMap.ToolTipSettings>
+			<telerik:HeatMapToolTipSettings>
+				<telerik:HeatMapToolTipSettings.ToolTipTemplate>
+					<DataTemplate>
+						<StackPanel Orientation="Horizontal" Background="White">
+							<TextBlock Text="Value: " FontSize="12"/>
+							<TextBlock Text="{Binding Value}" FontSize="12"/>
+						</StackPanel>
+					</DataTemplate>
+				</telerik:HeatMapToolTipSettings.ToolTipTemplate>
+			</telerik:HeatMapToolTipSettings>
+		</telerik:RadHeatMap.ToolTipSettings>
+		<telerik:RadHeatMap.Definition>
+			<telerik:CategoricalDefinition x:Name="categoricalDefinition" 
+									RowGroupMemberPath="Year" 
+									ColumnGroupMemberPath="Month" 
+									ValuePath="Temperature" />
+		</telerik:RadHeatMap.Definition>
+	</telerik:RadHeatMap>
+{{endregion}}
+
+#### __Figure 1: RadHeatMap with customized ToolTip__
+
+![RadHeatMap with selection enabled](images/radheatmap-tooltip_0.PNG)
```

Comment:
Can you include a pointing arrow cursor to the picture? This will make it more clear that a tooltip is displayed.